lib/model/content.dart

```diff
@@ -275,6 +272,22 @@ class ListNode extends BlockContentNode {
   }
 }
 
+class OrderedListNode extends ListNode {
+  const OrderedListNode({this.start = 1, required super.items, super.debugHtmlNode});
```

Comment:
nit: follow the patterns of the other classes nearby in the ordering of the named parameters